clients/java/signalr/src/main/java/com/microsoft/signalr/HubConnection.java

```diff
@@ -478,6 +508,10 @@ public void remove(String name) {
         logger.trace("Removing handlers for client method: {}.", name);
     }
 
+    /**
+     * Registers callbacks to run when the connection closes.
```

Comment:
Suggest: "Registers **a** callback~~s~~ to run when the connection **is** close**d**~~s~~."

The method only takes one callback at a time :)